cli/parser/parser.go

```diff
@@ -852,6 +852,11 @@ func consumeRCFileArgs(args []string, workspaceDir string) (newArgs []string, rc
 				encounteredDevNullBazelrc = true
 				continue
 			}
+
+			if !filepath.IsAbs(bazelrcArg) {
+				bazelrcArg = filepath.Join(workspaceDir, bazelrcArg)
+			}
```

Comment:
(since bazel can be invoked from a subdir of the workspace)

```suggestion
			if a, err := filepath.Abs(bazelrcArg); err == nil {
				bazelrcArg = a
			}
```

Reply:
Looks like this fails the test. I feel like we need the current working directory in there somehow, right?

Reply:
Oh looks like that’s what Abs does. Hmm…

Reply:
yeah good question, I wasn't actually sure how bazel deals with that.